cdhweb/context_processors.py

```diff
@@ -1,7 +1,7 @@
 from django.conf import settings
 from wagtail.core.models import Site
 
-from cdhweb.resources.utils import absolutize_url
+from cdhweb.pages.utils import absolutize_url
```

Comment:
can we check if this is still used anywhere? I think wagtail's `Page.get_full_url()` replaces it in most instances

Reply:
i can't read. this import proves that it is still used

Reply:
yeah, it's still used in a few places for non-page content.  — I wondered if there might be something in wagtail that obviated it, but thought it was ok to leave for now